pkgs/top-level/all-packages.nix

```diff
@@ -8279,6 +8279,8 @@ in
   bazel = callPackage ../development/tools/build-managers/bazel {
     inherit (darwin) cctools;
     inherit (darwin.apple_sdk.frameworks) CoreFoundation CoreServices Foundation;
+    buildJdk = jdk8;
+    buildJdkName = "jdk8";
```

Comment:
Seems a bit out of context here - it is not guaranteed that concating jdkX with this will result in a valid toolchain name. Could either pass in the full bazel target name for the toolchain, not expose it at all and derive from some version attribute of `buildJdk`, if that's possible?

That said, I don't see how different users of bazel-nix would want to configure their various toolchains, so it is hard to come up with any sensible parameter api.

Reply:
There’s is nothing to configure. This is just a codified reminder to change both the jdk and the toolchain when updating bazel.